### PR TITLE
Modify some visual elements to make them look more coherent

### PR DESCRIPTION
**Pull request by murtuzaalisurti**
Changed the following things:

1. The translation of the 'ayah' is now placed right under the arabic 'ayah' itself.
2. Gave title to the 'translation' and 'transliteration' of the Athkar. Redesigned it to look more coherent.
3. Did some minor UI changes.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -110,33 +117,29 @@ body {
...
 .translation-container .title {
   font-size: 20px;
-  background-color: #ccc;
-  width: fit-content;
-  border-radius: 10px 10px 0px 0px;
+  background-color: transparent;
+  width: 100%;
   padding: 5px 15px;
   font-weight: 600;
   z-index: 10;
```
Comment (shahednasser) [inline css/styles.css:134]: Can you remove this link as it now overlaps with the calendar when it opens

File: js/main.js [hunk 2]
```diff
@@ -97,11 +97,11 @@ $(document).ready(function(){
...
-  $(".translation-container").hover(function () {
-    $(this).children(".body").show('fast');
-  }, function () {
-    $(this).children(".body").hide('fast');
-  });
+  // $(".translation-container").hover(function () {
+  //   $(this).children(".body").show('fast');
+  // }, function () {
+  //   $(this).children(".body").hide('fast');
+  // });
```
Comment (shahednasser) [inline js/main.js:104]: You can remove these lines now

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @murtuzaalisurti thank you for your contribution and I'm sorry for taking long to check it out as I had a busy couple of weeks.

It all looks great! I just have a minor fixes below.

File: css/styles.css (hunk 1 above)
Comment (murtuzaalisurti) [reply css/styles.css]: removed the z-index property

File: js/main.js (hunk 2 above)
Comment (murtuzaalisurti) [reply js/main.js]: done

**Comment by shahednasser** [comment]
Great thanks! The changes will be a part of the next release

**Pull request merged**